builders/pallets-precompiles/precompiles/democracy.md

```diff
@@ -119,7 +125,16 @@ On the next screen, take the following steps:
 ![Get the encoded proposal](/images/builders/pallets-precompiles/precompiles/democracy/democracy-5.png)
 
 !!! note
-     You should NOT sign and submit the transaction here. You will submit this information via the **notePreimage** function in a later step.  
+     You should NOT sign and submit the transaction here. You will submit this information via the **notePreimage** function in the next step.  
+
+Now you can take the encoded proposal that you got from [Polkadot.js Apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fmoonbeam-alpha.api.onfinality.io%2Fpublic-ws#/democracy){target=_blank} and submit it via the `notePreimage` function of the democracy precompile. Despite its name, the preimage is not required to be submitted before the proposal. However, submitting the preimage is required before a proposal can be enacted. To submit the preimage via the `notePreimage` function, take the following steps:
```

Comment:
In the near future, I would like us to move to examples with Polkadot.js SDK when it makes sense. I think this is an end user oriented tutorial more than anything else, so it maybe makes sense to keep it as such, but not sure what you think